examples/merged/api/bar/baz.rs

```diff
@@ -15,7 +12,8 @@ pub async fn handler(_req: Request) -> Result<impl IntoResponse, Error> {
             json!({
               "message": format!("I choose you, {}!", starter),
             })
-            .to_string(),
```

Comment:
We could add something like this for ergonomics

```
impl From<Value> for Body {
    fn from(v: Value) -> Self {
        Body::Text(v.to_string())
    }
}
```

Reply:
Good shout. Not sure we can though, since `Body` isn't ours?